dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/categories/general/subcategories/production/LksgProduct.kt

```diff
@@ -0,0 +1,16 @@
+package org.dataland.datalandbackend.model.lksg.categories.general.subcategories.production
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * --- API model ---
+ * Fields of the LKSG questionnaire regarding a single product for the "Most Important Products" field
+ */
+data class LksgProduct(
+    @field:JsonProperty(required = true)
+    val productName: String,
+
+    val productionSteps: List<String>?,
+
+    val relatedCorporateSupplyChain: String?,
```

Comment:
Shouldn't this field be associated with a specific product step?

So maybe a new model that contains a specific production step as `string` and then the `relatedCorporateSupplyChain` as list of strings?

Reply:
I discussed the structure and our interpretation of the data dictionary with Siegfried and this is what we came up with.
If you have evidence that our interpretation is wrong, you can call me.

Reply:
Ok my evidence is the draw.io:

![image](https://github.com/d-fine/Dataland/assets/99255296/ce9d536e-6485-416c-9ffa-b769dcec6552)

There you get the impression that `relatedCorporateSupplyChain` info is attached to one specific production step.
If this is intended, then `relatedCorporateSupplyChain` should be inserted into the model per `productionStep`, and not independently.

Call me if you think we should further discuss this.